Makefile

```diff
@@ -30,9 +30,14 @@ OSTRICH_SRC  = $(CURDIR)/src/OSTRICH
 OSTRICH_TARGET = GCC    # can be also MPI but requires mpi compiler; not tested
 UNAME_S := $(shell uname -s)
 DOWNLOAD_CACHE = /tmp/
-FLYINGPIGEON_WPS_URL = http://localhost:8093
-FINCH_WPS_URL = http://localhost:5000
 
+# Additional servers used by notebooks
+FLYINGPIGEON_WPS_URL = https://pavics.ouranos.ca/twitcher/ows/proxy/flyingpigeon/wps
+FINCH_WPS_URL = https://pavics.ouranos.ca/twitcher/ows/proxy/finch/wps
+
+# Replace by these to run local servers
```

Comment:
Can also be given directly on the make commandline `make FLYINGPIGEON_WPS_URL=http://localhost:8093 FINCH_WPS_URL=http://localhost:5000 test-notebooks`, add a note so user is not forced to change the `Makefile` and potentially committing it accidentally.